cacti/datadog_checks/cacti/cacti.py

```diff
@@ -105,14 +107,16 @@ def _get_config(self, instance):
 
         return Config(host, user, password, db, rrd_path, whitelist, field_names, tags)
 
-    def _get_rrd_info(self, rrd_path):
+    @staticmethod
+    def _get_rrd_info(rrd_path):
         return rrdtool.info(rrd_path)
 
-    def _get_rrd_fetch(self, rrd_path, c, start):
+    @staticmethod
+    def _get_rrd_fetch(rrd_path, c, start):
         return rrdtool.fetch(rrd_path, c, '--start', str(start))
 
     def _read_rrd(self, rrd_path, hostname, device_name, tags):
-        ''' Main metric fetching method '''
+        """ Main metric fetching method."""
```

Comment:
Should that be a rule for the linter ?

Reply:
@ofek Any reason why string normalization is is disabled ? 

https://github.com/DataDog/integrations-core/blob/87278a286edf11279e55b6a9646f0cd4174a983e/pyproject.toml#L36

Reply:
So we can use single quotes for strings